lib/campaign.js

```diff
@@ -22,7 +22,7 @@ let _ = require('lodash')
     }]
   ]
   , unsupportedSyntax = [
-    [/\[Unsubscribe\]/i, 'Note: [Unsubscribe] tag was translated to <a href="?" data-msys-unsubscribe="1">. Remember to set that href tag correctly.'],
+    [/\[Unsubscribe\]/i, 'Note: [Unsubscribe] tag was translated to &lt;a href="?" data-msys-unsubscribe="1"&gt;...\nRemember to set that href tag correctly.'],
```

Comment:
ah, that rendered correctly in github. here is the screenshot, how it looks like in UI
<img width="985" alt="screen shot 2017-05-25 at 11 29 05 am" src="https://cloud.githubusercontent.com/assets/14877079/26457237/71c30f22-413d-11e7-9bc4-0b251e0855cb.png">